scrapy/pipelines/media.py

```diff
@@ -211,7 +211,7 @@ def _cache_result_and_execute_waiters(
             # minimize cached information for failure
             result.cleanFailure()
             result.frames = []
-            result.stack = None
+            result.stack = []
```

Comment:
Can you please revert this change? It's unrelated and has its own PR.

Reply:
done